docs/modules/ROOT/pages/metrics.adoc

```diff
@@ -0,0 +1,1678 @@
+
+[cols="2,1,4a"]
+.Metrics
+|===
+| Metric Name
+| Unit
+| Description
+
+|`cache.averageGetTime`
+|µs
+|the mean time to execute gets on the cache
+
+|`cache.averagePutTime`
+|µs
+|the mean time to execute puts on the cache
+
+|`cache.averageRemovalTime`
+|µs
+|the mean time to execute removes on the cache
+
+|`cache.cacheEvictions`
+|count
+|the number of evictions on the cache
+
+|`cache.cacheGets`
+|count
+|the number of gets on the cache
+
+|`cache.cacheHits`
+|count
+|the number of hits (successful get operations) on the cache
+
+|`cache.cacheHitPercentage`
+|percent
+|the percentage of hits (successful get operations) on the cache
+
+|`cache.cachePuts`
+|count
+|the number of puts to the cache
+
+|`cache.cacheRemovals`
+|count
+|the number of removals from the cache
+
+|`cache.cacheMisses`
+|count
+|the number of missed cache accesses on the cache
+
+|`cache.cacheMissPercentage`
+|percent
+|the percentage of missed cache accesses on the cache
+
+|`cache.creationTime`
+|ms
+|the creation of the cache on the member
+
+|`cache.lastAccessTime`
+|ms
+|the last access time to the cache
+
+|`cache.lastUpdateTime`
+|ms
+|the last update time to the cache
+
+|`cache.ownedEntryCount`
+|count
+|the owned entry count in the cache
+
+|`classloading.loadedClassesCount`
+|count
+|the number of classes that are currently loaded
+
+|`classloading.totalLoadedClassesCount`
+|count
+|the total number of classes that have been loaded since the instance has started execution.
+
+|`classloading.unloadedClassesCount`
+|count
+|the total number of unloaded classes.
+
+|`client.endpoint.count`
+|count
+|the number of active client endpoints for this member
+
+|`client.endpoint.totalRegistrations`
+|count
+|the total number of client endpoint registrations
+
+|`cluster.clock.clusterStartTime`
+|ms
+|the start time of the cluster (when the first node in cluster becomes master, its localClockTime value is saved as clusterStartTime)
+
+|`cluster.clock.clusterTime`
+|ms
+|the elapsed time since the cluster was created
+
+|`cluster.clock.clusterTimeDiff`
+|ms
+|the difference between this member's local clock timestamp and the cluster clock timestamp sent by the master node
+
+|`cluster.clock.clusterUpTime`
+|ms
+|the uptime for the cluster (current time - cluster start time)
+
+|`cluster.clock.localClockTime`
+|ms
+|this member's local clock timestamp
+
+|`cluster.clock.maxClusterTimeDiff`
+|ms
+|the maximum observed cluster time difference
+
+|`cluster.connection.closedTime`
+|count
+|the connection close time for this connection
+
+|`cluster.connection.connectionId`
+|count
+|the connection id for this client connection
+
+|`cluster.connection.eventHandlerCount`
+|count
+|the number of event handler for this client connection
+
+|`cluster.heartbeat.lastHeartbeat`
+|ms
+|the last timestamp of sending heartbeats to the other known cluster members
+
+|`cluster.size`
+|count
+|the size of the cluster
+
+|`cp.atomiclong.value`
+|count
+|the value of this AtomicLong
```

Comment:
Maybe making AtomicLong as monospaced as in the other primitives mentioned below, i.e., `IAtomicLong`